travis-setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euo pipefail
+
+# Sets up travis-ci environment for testing bioconda-utils.
+
+if [[ $TRAVIS_OS_NAME = "linux" ]]
+then
+    tag=Linux
+else
+    tag=MacOSX
+fi
+curl -O https://repo.continuum.io/miniconda/Miniconda3-latest-${tag}-x86_64.sh
+sudo bash Miniconda3-latest-${tag}-x86_64.sh -b -p /anaconda
+sudo chown -R $USER /anaconda
+export PATH=/anaconda/bin:$PATH
+
+# TODO: add remaining pip reqs to conda-forge
+conda install -y --file conda-requirements.txt --channel bioconda --channel conda-forge --channel anaconda
+
+python setup.py install
+
+pip install -r pip-test-requirements.txt
+pip install -r pip-requirements.txt
+
+mkdir -p /anaconda/conda-bld/osx-64 # workaround for bug in current conda
+mkdir -p /anaconda/conda-bld/linux-64 # workaround for bug in current conda
+conda index /anaconda/conda-bld/linux-64 /anaconda/conda-bld/osx-64
```

Comment:
I had tried taking them out but got failures, that may have been pre-conda-build-2.0 though. Will try again.